prow/plugins/cat/cat.go

```diff
@@ -137,6 +138,9 @@ func (c *realClowder) URL(category string, movieCat bool) string {
 }
 
 func (c *realClowder) readCat(category string, movieCat bool) (string, error) {
+	if category == "no" {
+		return grumpyURL, nil
```

Comment:
This will result in a text link in the GitHub comment instead of actually embedding the image. The `catResult.Format()` method only uses the `cr.Image` field so it could be switched to a function like `func Format(url string) (string, error)` and used here.

It would also be good practice to refactor the `readCat` method so that we do the size check from line 164 for any url we might return including the `grumpyURL`.